ConfigurationSystem/Agent/Bdii2CSAgent.py

```diff
@@ -301,7 +305,7 @@ def __updateCS(self, bdiiChangeSet):
         queueVODict[section] = queueVODict[section].union(set(new_value.split(',')))
       else:
         changeSet.add(entry)
-    for section, VOs in queueVODict.items():
+    for section, VOs in queueVODict.iteritems():
```

Comment:
According to the Python 3 migration plan, I guess this should stay unchanged

Reply:
Done.